docs/content/console.md

```diff
@@ -0,0 +1,94 @@
+# CLI tool
+
+DbTools ships a standalone console tool for running it as an standalone application.
+
+If installed via composer, simply run:
+
+```sh
+./vendor/bin/db-tools
+```
+
+This CLI tool runs outside of the framework and application context, therefore requires
+that you provide a dedicated configuration file for it.
+
+:::info
+A script that generates a working PHAR file is provided in the repository and can be
+used to create a self-contained PHAR archive for running in a docker container, on
+a baremetal with PHP installed, etc...
+
+Future plans are to provide an official PHAR archive for each stable release.
+:::
+
+## Configuration
+
+### Configuration file
+
+The configuration file is a `YAML` file whose available options are strictly identical
+to the Symfony bundle configuration with a few additional parameters dedicated to the
+standalone application.
+
+:::tip
+Please refer to the [configuration reference](configuration/reference) for a complete
+and detailled configuration options list.
+:::
+
+The most important and required one is the list of available database connections,
+using the `connections` configuration option.
+
+```yaml [Standalone]
+connections:
+    connection_one: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
+    connection_two: "mysql://username:password@hostname:port?version=8.1&other_option=..."
+```
+
+Keys are connection names for idenfying those as a command line option, values are
+database URL containing all necessary information and options for connecting.
```

Comment:
May be you can start by documenting when there is only one connection (which will be most cases) and then specify that you can provide multiple

Reply:
I though I fixed this, I may have broke it during merge. I'll rewrite this part of the documentation.